app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,12 @@
+class MovieActorsController < ApplicationController
+
+  def new
```

Comment:
The "meat" of this method is great - I think it is better called `create` instead of `new` though. In Rails, `new` is understood to be an action that shows a form that _leads to_ a `create` method. Otherwise good!